plugins/identity/app/controllers/identity/domains/technical_users_controller.rb

```diff
@@ -4,17 +4,23 @@ module Identity
   module Domains
     class TechnicalUsersController < ::DashboardController
 
+      before_filter do
+        @policy_default_params ||= {}
+        @policy_default_params[:user] = { domain_id: @scoped_domain_id }
```

Comment:
why is the user relevant at all in the policy? shouldn't it just check for other cloud_admin or a admin role for the current domain?